### DISCUSSION TITLE
Pangesture Behaves unexpected when using context value to the useAnimatedGestureHandler its just disappear the view any help

### DISCUSSION THREAD

**Discussion by vikil143** [software-mansion/react-native-reanimated#2072] [Ideas]
`

export default function BottomSheet({show, hide}: BottomSheetProps) {
  const dragY = useSharedValue(0);

  const panHandler = useAnimatedGestureHandler<
    PanGestureHandlerGestureEvent,
    {y: number}>({
    onStart: (_event, {y}) => {
      y = dragY.value;
    },
    onActive: ({translationY}, ctx) => {
      dragY.value = translationY + ctx.y;
    },
  });

 

  const wrapperStyle = useAnimatedStyle(() => {
    return {
      transform: [{translateY: dragY.value}],
    };
  });

  return (
    <Animated.View style={[styles.container]}>
      <PanGestureHandler onGestureEvent={panHandler}>
        <Animated.View style={[styles.wrapper, wrapperStyle]}></Animated.View>
      </PanGestureHandler>
    </Animated.View>
  );
}

const styles = StyleSheet.create({
  wrapper: {
    flex: 1,
    backgroundColor: 'red',
  },
  container: {
    ...StyleSheet.absoluteFillObject,
    flex: 1,
    width: '100%',
    height: '100%',
    zIndex: 10,
  },
});
`

**Comment by NikitaDudin** [comment]
```
onStart: (_event, {y}) => {
y = dragY.value;
},
```
don't use destructuring for ctx in parameters.
 ```
onStart: (_event, ctx) => {
ctx.y = dragY.value;
},
```

**Reply by vikil143** [reply]
Any reason behind this

**Comment by vikil143** [comment]
@NikitaDudin Thanks now its working
